src/map.cpp

```diff
@@ -4729,11 +4729,11 @@ item &map::add_item_at( const tripoint &p,
 item map::water_from(const tripoint &p)
 {
     if( has_flag( "SALT_WATER", p ) ) {
-        item ret( "salt_water", 0 );
+        item ret( "salt_water", 0, std::numeric_limits<long>::max() );
```

Comment:
``src/map.cpp:4732:36: warning: implicit conversion from 'type' (aka 'long') to 'int' changes value from 9223372036854775807 to -1 [-Wconstant-conversion]``